WINDOWS.md

```diff
@@ -471,7 +471,7 @@ Then, let's disable warning for copy-pasting commands between Windows and Ubuntu
 
 :warning: Do not forget the comma at the end of the line!
 
-You can save these changes by pressing `CTRL` + `V`
+You can save these changes by pressing `CTRL` + `S`
```

Comment:
Does that comes from a `lewagon/setup` partial?

Reply:
yes, looks like a github action 👌

Reply:
yes, looks like this was injected by a github action from the web partial `setup/windows_terminal`